src/TimeIntegration.cpp

```diff
@@ -175,18 +175,22 @@ void TimeIntegration::run() {
             // RMS error at midposize_t at each time step Nt
             analysis->calc_L2_norm_mid_point(m_solver, t_cur, Sum);
 
-            // To check CFL and VN, comment out
-            /*bool CFL_check = ana.check_time_step_CFL(u, v, w, dt);
-            if(!CFL_check){
-                std::cout<<"CFL condition not met!\n";
-                // To change dt, comment out
-                //dt = ana.set_DT_with_CFL(u, v, w);
-                //std::cout<<" Setting dt = "<<dt<<std::endl;
+            // check CFL
+            real cfl = analysis->calc_CFL(u, v, w, dt);
+
+            // CFL condition not met adapting dt
```

Comment:
there isn't an adaption of dt, I propose to change the comment into `CFL condition not met, automatic dt correction can be considered by setting dt = `... ?

Reply:
adapted comment in code in bfb9d53